modules/nodes-descheduler-strategies.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-strategies_{context}"]
+= Descheduler strategies
+
+The following descheduler strategies are available:
+
+Low node utilization::
+The `lownodeutilization` strategy finds nodes that are underutilized and evicts Pods, if possible, from other nodes in the hope that recreation of evicted Pods will be scheduled on these underutilized nodes.
++
+The underutilization of nodes is determined by several configurable threshold parameters: CPU, memory, and number of Pods. If a node's usage is below the configured thresholds for all parameters (CPU, memory, and number of Pods), then the node is considered to be underutilized.
++
+You can also set a target threshold for CPU, memory, and number of Pods. If a node's usage is above the configured target thresholds for all parameters, then the node's Pods might be considered for eviction.
++
+Additionally, you can set the strategy to activate only when the number of underutilized nodes is above the configured value. This can be helpful in large clusters where a few nodes might be underutilized frequently or for a short period of time.
+
+Duplicate Pods::
+The `duplicates` strategy ensures that there is only one Pod associated with a ReplicaSet, ReplicationController, Deployment, or Job running on same node. If there are more, then those duplicate Pods are evicted for better spreading of Pods in a cluster.
++
+This situation could occur after a node failure, when a Pods is moved to another node, leading to more than one Pod associated with a Deployment, for example, on that node. After the failed node is ready again, this strategy evicts the duplicate Pod.
```

Comment:
NIT: I'd remove "for example" in this line.

Reply:
@lamek the reason I have that is because it doesn't necessarily have to be associated with a Deployment. It could be a ReplicaSet, ReplicationController, or Job potentially. I'll think on it, but let me know if you have a better suggestion to make this clearer w/o the weird for example in the middle :).

Reply:
I found a way to reword it that I'm happy with :)